src/pages/BasketPage/components/BasketPageContent/index.js

```diff
@@ -1,141 +1,221 @@
-          itemsList={useBasketProducts.data}
-          filtredCategories={useProductsCategories.data}
-        />
-        <EditModeList
-          parentsTitle="Dishes"
-          itemsList={useBasketDishes.data}
-          filtredCategories={dishesCategories.data}
-        />
-        <EditModeList
-          parentsTitle="SavedList"
-          itemsList={useBasketSavedLists.data}
-          filtredCategories={useSavedListsCategories.data}
-        />
-        <Switch>
-          <Route path="/basket/createBasketHistory">
-            <Modal>
-              <CreateHistoryModalContent />
-            </Modal>
-          </Route>
-        </Switch>
-      </div>
-    ) : (
-      <div>
-        <ListModeList
-          parentsTitle="Groceries"
-          itemsList={useBasketGroceres.data}
-          filtredCategories={useGroceriesCategories.data}
-        />
-        <ListModeList
-          parentsTitle="Products"
-          itemsList={useBasketProducts.data}
-          filtredCategories={useProductsCategories.data}
-        />
-        <ListModeList
-          parentsTitle="Dishes"
-          itemsList={useBasketDishes.data}
-          filtredCategories={dishesCategories.data}
-        />
-        <ListModeList
-          parentsTitle="SavedList"
-          itemsList={useBasketSavedLists.data}
-          filtredCategories={useSavedListsCategories.data}
-        />
-      </div>
-    )
+  const groceriesCategoriesData = useSelector(selectGroceriesCategoriesData)
+  const groceriesBasketData = useSelector(selectGroceriesBasketData)
+  const GroceriesStatus = useSelector(selectGroceriesStatus)
+
+  const productsCategories = useSelector(selectProductsCategoriesData)
+  const productsBasketData = useSelector(selectProductsBasketData)
+  const productsStatus = useSelector(selectProductsStatus)
+
+  const dishesCategories = useSelector(selectDishesCategoriesData)
+  const dishesBasketData = useSelector(selectDishesBasketData)
+  const dishesStatus = useSelector(selectDishesStatus)
+
+  const savedListCategories = useSelector(selectSavedListCategoriesData)
+  const savedListBasketData = useSelector(selectSavedListBasketData)
+  const savedListStatus = useSelector(selectSavedListStatus)
+
+  const parentCategories = useSelector(selectParentCategoriesData)
+  const parentCategoriesStatus = useSelector(selectParentCategoriesStatus)
+
+  const data = dataLoadingStatus([
+    GroceriesStatus,
+    productsStatus,
+    dishesStatus,
+    savedListStatus,
+    parentCategoriesStatus,
+  ])
+
+  if (data.isError) {
+    return 'Fetching data error...'
+  }
+
+  if (data.isLoading) {
+    return 'Loading data...'
   }
+
+  if (!data.isLoaded) {
+    return null
+  }
+  const basketEmpty =
+    groceriesBasketData.length === 0 &&
+    productsBasketData.length === 0 &&
+    dishesBasketData.length === 0 &&
+    savedListBasketData.length === 0
+
+  const messege = <Styles.Message>Your basket is empty!</Styles.Message>
+
+  return basketEmpty ? (
+    messege
```

Comment:
po prostu daj tu <Styles.Message>Your basket is empty!</Styles.Message>

Reply:
Done.